baselines/nadamw/pytorch/submission.py

```diff
@@ -191,23 +193,20 @@ def init_optimizer_state(workload: spec.Workload,
           NAdamW(
               model_params.parameters(),
               lr=hyperparameters.learning_rate,
-              betas=(hyperparameters.beta1, hyperparameters.beta2),
+              betas=(1.0 - hyperparameters.one_minus_beta1,
+                     hyperparameters.beta2),
               eps=1e-8,
               weight_decay=hyperparameters.weight_decay)
   }
 
   def pytorch_cosine_warmup(step_hint: int, hyperparameters, optimizer):
```

Comment:
is it possible to just use optax schedules everywhere ? not sure if it works but if it does that's be a good thing to do so jax and pytorch submissions don't have different schedules

Reply:
leaving as an exercise for a future PR :)